test/functional/p2p_ibd_txrelay.py

```diff
@@ -24,6 +24,7 @@ def set_test_params(self):
         ]
     def run_test(self):
         self.log.info("Check that nodes set minfilter to MAX_MONEY while still in IBD")
+        self.sync_all()
```

Comment:
There are no blocks or transactions to sync, so sync_all should be a no-op.

I presume the issue is that the send messages loop for the peer hasn't been called (yet). This could be fixed by forcing a ping (with the RPC ping) and then waiting for it to propagate, I think.

Reply:
Good point, thanks! Trying a simple solution with c19310c.